client/src/views/Feed/SinglePost.vue

```diff
@@ -3,8 +3,15 @@
     <v-comments :isModelVisible="isModelVisible" :closePopup="closePopup" :id="post.postId" />
     <div class="video-container" @click="play">
       <figure class="image is-32x32">
-        <span class="emoji video-emoji" v-html="post.emoji" />
-        <avatar :img="post.avatarUrl" :size="'30px'" />
+        <span class="emoji video-emoji" v-html="post.emoji"/>
+        <router-link :to="{ path: `/profile/${post.nickname}` }">
```

Comment:
I'd suggest moving "/profile/${post.nickname" into computed since it's used multiple times.
But this is just a suggestion for future. No need to update this now. Merging as is.